hoon/reference/irregular.md

```diff
@@ -151,44 +162,67 @@ Hoon has the usual branches and logical tests.
 [docs](@/docs/hoon/reference/rune/wut.md#wutzap) \\!
 `[%wtzp p=hoon]`: logical not.
 
-R: `?!(p)`
-I: `!(p)`
+Regular: `?!(p)`
+
+Irregular: `!(p)`
 
 ### ?& wutpam
 
 [docs](@/docs/hoon/reference/rune/wut.md#wutpam) \\&
 `[%wtpm p=(list hoon)]`: logical and.
 
-R: `?&(p)`
-I: `&(p)`
+Regular: `?&(p)`
+
+Irregular: `&(p)`
 
 ### ?| wutbar
 
 [docs](@/docs/hoon/reference/rune/wut.md#wutbar) \\|
 `[%wtbr p=(list hoon)]`: logical or.
 
-R: `?|(p)`
-I: `|(p)`
+Regular: `?|(p)`
+
+Irregular: `|(p)`
 
 ## ^ ket (cast)
 
 Lets us adjust types without violating type constraints.
 
+### ^: ketcol
+
+[docs](@/docs/hoon/reference/ket.md#ketcol) \\,
```

Comment:
These lines that start with `[docs]` should have another space after them to avoid inlining with the next line.

Reply:
Good catch